pkg/types/validation/installconfig_test.go

```diff
@@ -1930,6 +1930,20 @@ func TestValidateInstallConfig(t *testing.T) {
 			}(),
 			expectedError: "platform.openstack.apiVIPs: Invalid value: \"foobar\": \"foobar\" is not a valid IP",
 		},
+		{
+			name: "should reject load balancer on OpenStack if not TechPreviewNoUpgrade",
+			installConfig: func() *types.InstallConfig {
+				c := validInstallConfig()
+				c.Platform = types.Platform{
+					OpenStack: validOpenStackPlatform(),
+				}
+				c.Platform.OpenStack.LoadBalancer = &configv1.OpenStackPlatformLoadBalancer{
+					Type: configv1.LoadBalancerTypeOpenShiftManagedDefault,
+				}
+				return c
+			}(),
+			expectedError: `platform.openstack.loadBalancer: Forbidden: the TechPreviewNoUpgrade feature set must be enabled to use this field`,
```

Comment:
Please do not test OpenStack's platform-specific logic here. We have our check in a separate directory so that we can operate autonomously on platform-specific fields; if the test is in the shared logic, it means that we again need Installer-team's approval for a platform-specific matter.